apis/cc-de_CH/atrium-api-cc-de_CH-common/src/main/kotlin/ch/tutteli/atrium/api/cc/de_CH/mapAssertions.kt

```diff
@@ -6,6 +6,15 @@ import ch.tutteli.atrium.creating.AssertionPlant
 import ch.tutteli.atrium.creating.AssertionPlantNullable
 import ch.tutteli.atrium.domain.builders.AssertImpl
 
+/**
+ * Makes the assertion that [AssertionPlant.subject]'s [Map] contains [key].
```

Comment:
I would rephrase to 
`Makes the assertion that [AssertionPlant.subject] contains the given [key].`